concepts/extensions/apps-concept.md

```diff
@@ -27,3 +27,7 @@ Notice that this is optional in the case that Shopware and your app don't need t
 
 Your app can modify the appearance of the storefront, by shipping your storefront assets \(template files, javascript sources, SCSS sources, snippet files\) alongside your manifest file. You don't need to serve those assets from your external server, as Shopware will build the storefront anew on installation of your app and will consider your modifications in that process. Find out more about modifying the appearance of the storefront in the App Storefront Guide.
 {% page-ref page="../../guides/plugins/apps/app-base-guide.md" %}
+
+## Integrate payment providers
```

Comment:
Rather put this into the above paragraph or rephrase, as this is a copy of the first paragraph of guide.

Reply:
rephrased it :)